.github/workflows/dotnet-desktop.yml

```diff
@@ -0,0 +1,56 @@
+name: .Net Core Desktop
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build-test:
+    name: Build and Test
+    runs-on: windows-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.0
+
+    - name: Setup MSBuild.exe
+      uses: microsoft/setup-msbuild@v2
+
+    - name: Install ntools and dev tools
```

Comment:
@ahmdhjj No need to insyall all devtools.  The runner already has git and VS 2022. We just need to install ntools via PowerShell the same way we did in DevSetup.ps1

Reply:
@naz-hage done